lib/tasks/remove_students.rake

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative '../../app/services/student_removal_service'
+require 'csv'
+
+namespace :remove_students do
```

Comment:
I would suggest adding a "dry run" flag to this task so that it can be run, and report on its actions, without affecting any data.

Reply:
haha, I started with one, but when I factored it out into the service that became tricky - and I felt it wouldn't be doing much, it could at least say how many students it would affect?